### PR TITLE
methanol in shipping

### PR DESCRIPTION
**Pull request by fneum**

### CODE REVIEW

**Comment by fneum** [comment]
@euronion could you have a quick check that the efficiencies make sense?

File: config.default.yaml [hunk 1]
```diff
@@ -193,6 +193,10 @@ sector:
   agriculture_machinery_electric_share: 0
   agriculture_machinery_fuel_efficiency: 0.7 # fuel oil per use
   agriculture_machinery_electric_efficiency: 0.3 # electricity per use
+  shipping_methanol: true
```
Comment (euronion) [inline config.default.yaml:196]: Wouldn't it be better to make this like the `shipping_hydrogen_share` a `shipping_methanol_share`?

File: scripts/prepare_sector_network.py [hunk 2]
```diff
@@ -2154,6 +2164,31 @@ def add_industry(n, costs):
...
+            bus1=spatial.methanol.nodes,
+            bus2=nodes,
+            bus3=spatial.co2.nodes,
+            carrier="methanolisation",
+            p_nom_extendable=True,
+            capital_cost=costs.at["methanolisation", 'fixed'] * options["MWh_MeOH_per_MWh_H2"], # EUR/MW_H2/a
+            lifetime=costs.at["methanolisation", 'lifetime'],
+            efficiency=options["MWh_MeOH_per_MWh_H2"],
+            efficiency2=- options["MWh_MeOH_per_MWh_H2"] / options["MWh_MeOH_per_tCO2"],
+            efficiency3=- options["MWh_MeOH_per_MWh_H2"] / options["MWh_MeOH_per_MWh_e"],
```
Comment (euronion) [inline scripts/prepare_sector_network.py:2189]: `bus2` is electricity and `bus3` CO2, right?

Shouldn't then be `efficiency2` about MWh_e and `efficiency3` about CO2 (currently it is the other way around)

**Review by euronion — COMMENTED** [review]
Efficiencies look ok. Some comments, see below.

Question: Isn't there a connection from the methanol bus -> shipping demand (load) missing?

File: scripts/prepare_sector_network.py (hunk 2 above)
Comment (fneum) [reply scripts/prepare_sector_network.py:2189]: Yup, great catch!

**Pull request merged**